pthread_key_t definition missing in lib/std/c.zig

### Zig Version

0.11.0

### Steps to Reproduce and Observed Behavior

```const std = @import("std");

pub fn main() void {
    var key: std.c.pthread_key_t = undefined;
    _ = std.c.pthread_key_create(&key, null);
}
```

Compiling the above program results in the following error:
```
pthread_key_t-issue.zig:4:19: error: root struct of file 'c' has no member named 'pthread_key_t'
    var key: std.c.pthread_key_t = undefined;
             ~~~~~^~~~~~~~~~~~~~
referenced by:
    comptime_0: /home/rganesan/git/zig/build/stage3/lib/zig/std/start.zig:59:50
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

```

### Expected Behavior

Program should compile

--- a/lib/std/c.zig
+++ b/lib/std/c.zig
@@ -371,6 +371,7 @@ pub extern "c" fn pthread_rwlock_trywrlock(rwl: *c.pthread_rwlock_t) callconv(.C
 pub extern "c" fn pthread_rwlock_unlock(rwl: *c.pthread_rwlock_t) callconv(.C) c.E;
 
 pub const pthread_t = *opaque {};
+pub const pthread_key_t = *opaque {};
 pub const FILE = opaque {};
 
 pub extern "c" fn dlopen(path: [*:0]const u8, mode: c_int) ?*anyopaque;

